selfservice/strategy/oidc/strategy.go

```diff
@@ -114,6 +116,11 @@ type authCodeContainer struct {
 	Traits json.RawMessage `json:"traits"`
 }
 
+func generateState(flowID string) string {
+	state := x.NewUUID().String()
+	return base64.RawURLEncoding.EncodeToString([]byte(fmt.Sprintf("%s:%s", flowID, state)))
```

Comment:
@zepatrik fyi - encoding the flow ID + state so we can detect the flow ID without a cookie